docs/tutorials/monitor-hadoop-cluster.md

```diff
@@ -0,0 +1,179 @@
+    "name" : "Hadoop:service=DataNode,name=DataNodeActivity-dev-slave-01.dev.loc
+al-9866",
+    "modelerType" : "DataNodeActivity-dev-slave-01.dev.local-9866",
+    "tag.SessionId" : null,
+    "tag.Context" : "dfs",
+    "tag.Hostname" : "dev-slave-01.dev.local",
+    "BytesWritten" : 500960407,
+    "TotalWriteTime" : 463,
+    "BytesRead" : 80689178,
+    "TotalReadTime" : 41203,
+    "BlocksWritten" : 16,
+    "BlocksRead" : 16,
+    "BlocksReplicated" : 4,
+    ...
+  },
+  { ... }
+  ]
+}
+```
+
+If Netdata can't access the `/jmx` endpoint for either a NameNode or DataNode, it will not be able to auto-detect and
+collect metrics from your HDFS implementation.
+
+Zookeeper auto-detection relies on an accessible client port and a whitelisted `mntr` command. For more details on
+`mntr`, see Zookeeper's documentation on [cluster
+options](https://zookeeper.apache.org/doc/current/zookeeperAdmin.html#sc_clusterOptions) and [Zookeeper
+commands](https://zookeeper.apache.org/doc/current/zookeeperAdmin.html#sc_zkCommands).
+
+## Configure the HDFS and Zookeeper modules
+
+To configure Netdata's HDFS module, navigate to your Netdata directory (typically at `/etc/netdata/`) and use
+`edit-config` to initialize and edit your HDFS configuration file.
+
+```bash
+cd /etc/netdata/
+sudo ./edit-config go.d/hdfs.conf
+```
+
+At the bottom of the file, you will see two example jobs, both of which are commented out:
+
+```yaml
+# [ JOBS ]
+#jobs:
+#  - name: master
+#    url: http://127.0.0.1:9870/jmx
+#
+#  - name: slave
+#    url: http://127.0.0.1:9864/jmx
+```
+
+Uncomment these lines and edit the `url` value(s) according to your setup. Now's the time to add any other configuration
+details, which you can find inside of the `hdfs.conf` file itself. Most production implementations will require TLS
+certificates.
+
+The result for a simple HDFS setup, running entirely on `localhost` and without certificate authentication, might look
+like this:
+
+```yaml
+# [ JOBS ]
+jobs:
+  - name: master
+    url: http://127.0.0.1:9870/jmx
+
+  - name: slave
+    url: http://127.0.0.1:9864/jmx
+```
+
+At this point, Netdata should be configured to collect metrics from your HDFS servers. Let's move on to Zookeeper.
+
+Next, use `edit-config` again to initialize/edit your `zookeeper.conf` file.
+
+```bash
+cd /etc/netdata/
+sudo ./edit-config go.d/zookeeper.conf
+```
+
+As with the `hdfs.conf` file, head to the bottom, uncomment the example jobs, and tweak the `address` values according
+to your setup. Again, you may need to add additional configuration options, like TLS certificates.
+
+```yaml
+jobs:
+  - name    : local
+    address : 127.0.0.1:2181
+
+  - name    : remote
+    address : 203.0.113.10:2182
+```
+
+Finally, restart Netdata.
+
+```sh
+sudo service restart netdata
+```
+
+Upon restart, Netdata should recognize your HDFS/Zookeeper servers, enable the HDFS and Zookeeper modules, and begin
+showing real-time metrics for both in your Netdata dashboard. 🎉
+
```

Comment:
we could mention default alarms for hdfs/zookeeper

 - [hdfs alarms](https://github.com/netdata/netdata/blob/master/health/health.d/hdfs.conf)
 - [zookeeper](https://github.com/netdata/netdata/blob/master/health/health.d/zookeeper.conf)